Source/Csla/Server/SanitizedServerSideDataPortalException.cs

```diff
@@ -0,0 +1,31 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="SanitizedServerSideDataPortalException.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Sanitized exception for server-side data portal operations.</summary>
+//-----------------------------------------------------------------------
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Csla.Server
+{
+  /// <summary>
+  /// Sanitized server-side data portal exception; used to 
+  /// avoid the transmission of sensitive server-side information
+  /// to the client in remote data portal operations
+  /// </summary>
+  public class SanitizedServerSideDataPortalException : ApplicationException
```

Comment:
I think maybe just `ServerException` would be a better name? Short and to the point?

It'll end up in a `Csla.DataPortalException` on the client in any case - so just indicating that its origin is a general server exception (like a 500 http result) seems reasonable.

Reply:
Yes, it's certainly a bit long, now you mention it!

I agree, ServerException would work. However, I do wonder if that would be cybersquatting, in the sense that I would be using a name that might be useful for something else.

You'll note that internally this type uses a resource string which makes its function very specific. It's not really all that reusable. I suppose i could change that so that the message is passed in, but the token replacement that is performed does mean that the format of the string is important.

SanitizedException is an alternative I suppose.

I'll think on it for a few minutes before diving in and making the change.